gordo_components/dataset/base.py

```diff
@@ -54,11 +55,23 @@ def join_timeseries(
         resolution
             The bucket size for grouping all incoming time data (e.g. "10T")
 
+        aggregation_methods
+            Aggregation method(s) to use for the resampled buckets. If a single
+            resample method is provided then the resulting dataframe will have names
+            identical to the names of the series it got in. If several
+            aggregation-methods are provided then the resulting dataframe will
+            have a multi-level index, with the first level being the series-name
```

Comment:
Nit-pick alert: :alarm_clock:  I would say `multi-level column index` or something like you say in the `Returns` section below.